app/Application/Services/OpenWalletService.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Application\Services;
+
+use App\Application\DataSources\UserDataSource;
```

Comment:
If we are working with hexagonal architecture, these DataSources its good located in the Application layer?